changes/485.feature.rst

```diff
@@ -0,0 +1 @@
+Added new nirspec_dark model with dark_rate and dark_rate_unc and added p_gratin to dark and superbias models
```

Comment:
I think this could be split onto two lines, something like
```
Added new DarkNirspecModel that includes `dark_rate` and `dark_rate_unc` extensions.
Added `p_gratin` header keyword to dark and superbias models
```
and if it's easy to explain in half a sentence, you could add the reason why `p_gratin` is being added.  Isn't it about CRDS queries?